src/sentry/api/event_search.py

```diff
@@ -269,7 +277,16 @@ def visit_quoted_raw_search(self, node, children):
             return None
         return SearchFilter(SearchKey('message'), "=", SearchValue(value))
 
-    def visit_numeric_filter(self, node, (search_key, _, operator, search_value)):
+    def visit_boolean_term(self, node, children):
```

Comment:
It might make sense to put this behind a feature switch. Because if someone does an and/or search now, it'll throw an error on the event search pages since nothing understands what `SearchBoolean` is yet.

Not sure on the best way to do this. I wonder if we should generate the parser grammar in a method so that you can just leave the boolean logic out if we don't want to use it?

Reply:
Yes there's https://github.com/getsentry/sentry/pull/12970 to address the flag. Let's move this conversation to that pr. :) Also interesting idea about the method.

Reply:
Nevermind my last comment. I'm going to combine the two. It's a bit too hard to do them separately.